packages/components/src/styles/components/stepper/list.scss

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// STEPPER > LIST
+//
+
+$hds-stepper-list-step-padding-bottom: 16px;
+
+.hds-stepper-list,
+.hds-stepper-list__step {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+// SUB COMPONENT: STEPPER > LIST > STEP
+
+.hds-stepper-list__step {
+  display: grid;
+  grid-column-gap: 8px;
+  grid-template-columns: auto 1fr;
+
+  &:not(:last-of-type) {
+    padding-bottom: $hds-stepper-list-step-padding-bottom;
```

Comment:
If you use flexbox & gap, you can remove this.

(Btw if not using flexbox or CSS grid + `gap` for spacing, using `margin` is preferable to `padding` in most cases for setting spacing between elements.)